libtsuba/include/tsuba/AsyncOpGroup.h

```diff
@@ -3,24 +3,27 @@
 
 #include <future>
 #include <list>
+#include <optional>
 
 #include "katana/Result.h"
 
 namespace tsuba {
 
+using AsyncError = std::optional<katana::CopyableErrorInfo>;
```

Comment:
I'm wondering if we should just have a `CopyableResult` which is like a `Result` but uses a string for error info and we can remove `CopyableErrorInfo` since it hasn't really caught on.

Reply:
yes, sorry; that is what I should have said

Reply:
Sounds good. I'll aim to do that instead.

Reply:
It's less trivial than I thought at the beginning. This is going to take me some time since I have other things going on.